anitya/lib/backends/maven.py

```diff
@@ -70,3 +75,96 @@ def get_versions(cls, project):
                           .replace(':', '/'))
 
         return get_versions_by_regex(url, REGEX, project)
+
+    @classmethod
+    def verify_url(cls, group_id, artifact_id):
```

Comment:
This needs a documentation block in the ["google"-style](http://www.sphinx-doc.org/en/1.5.5/ext/example_google.html).